rust/agents/relayer/src/merkle_tree/processor.rs

```diff
@@ -76,7 +76,7 @@ impl MerkleTreeProcessor {
                 .set(insertion.index() as i64);
             Some(insertion)
         } else {
-            debug!(leaf_index=?self.leaf_index, "No message found in DB for leaf index");
+            trace!(leaf_index=?self.leaf_index, "No more messages found in DB for leaf index. Done processing.");
```

Comment:
nit: this kinda reads to me like there are multiple messages for a single leaf index and that the task is done, maybe something like "No merkle tree insertion found in DB for leaf index, waiting for it to be indexed" or something?

Reply:
ah yes good point